### PR TITLE
*: Update the owner timeout of background job and pass golint

### PR DESCRIPTION
**Pull request by zimulala**
ddl: update the timeout of background job checking owner
        background job is serial processing, so we can extend the owner timeout to make sure a batch of task will be processed before timeout.
util: pass golint

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @shenli @coocood

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -161,6 +162,10 @@ func asyncNotify(ch chan struct{}) {
 	}
 }
 
+// Background job is serial processing, so we can extend the owner timeout to make sure
+// a batch of task will be processed before timeout.
```
Comment (shenli) [inline ddl/ddl_worker.go]: task -> tasks  or task -> rows

File: ddl/bg_worker.go [hunk 2]
```diff
@@ -150,7 +150,8 @@ func (d *ddl) onBackgroundWorker() {
 
 	// we use 4 * lease time to check owner's timeout, so here, we will update owner's status
 	// every 2 * lease time, if lease is 0, we will use default 10s.
-	checkTime := chooseLeaseTime(2*d.lease, 10*time.Second)
+	invalid := time.Duration(0)
+	checkTime := chooseLeaseTime(invalid, 2*d.lease, 10*time.Second)
```
Comment (shenli) [inline ddl/bg_worker.go]: Should we update the comments above?

Comment (zimulala) [inline ddl/bg_worker.go]: Update comments for invalid？

Comment (shenli) [inline ddl/bg_worker.go]: Yes，or add comment for the parameter of chooseLeaseTime.

**Comment by zimulala** [comment]
PTAL @shenli @coocood

File: ddl/ddl_worker_test.go [hunk 3]
```diff
@@ -84,6 +85,9 @@ func (s *testDDLSuite) TestCheckOwner(c *C) {
 	testCheckOwner(c, d2, true, ddlJobFlag)
 	testCheckOwner(c, d2, true, bgJobFlag)
 
+	time.Sleep(20 * time.Second)
```
Comment (coocood) [inline ddl/ddl_worker_test.go]: Sleep too long for a test.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**